src/main/java/Unit/Interface/Move.java

```diff
@@ -6,13 +6,13 @@
 
     default void moveTo(Unit unit, int x, int y){
         if (unit instanceof GroundUnit){
-            unit.x = x;
-            unit.y = y;
-            System.out.println(unit.nickName+" : 지상이동 to Location("+x+", "+y+")");
+            unit.setX(x);
+            unit.setY(y);
```

Comment:
```suggestion
            unit.move(x, y);
```

setX, setY 의 경우 해당 유닛의 좌표값을 이동시키는것 같은데
setXXX 와 같은 세터 기반의 함수 사용은 지양하도록 하고, 조금 더 추상적인 메서드 네이밍을 고려해 만들도록 합시다.

해당 유닛이 취할 수 있는 행위자체를 메서드로 만들어서 사용해주세요.